eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Newpp.hs

```diff
@@ -108,7 +108,7 @@ newPpTransition = do
         == utxosDeposited utxoSt
           ?! UnexpectedDepositPot (Coin oblgCurr) (utxosDeposited utxoSt)
 
-      if (ppNew' ^. ppMaxTxSizeL + ppNew' ^. ppMaxBHSizeL) < (ppNew' ^. ppMaxBBSizeL)
+      if (ppNew' ^. ppMaxTxSizeL + fromIntegral (ppNew' ^. ppMaxBHSizeL) < ppNew' ^. ppMaxBBSizeL)
```

Comment:
Although this is not really possible, since we would never submit such high values in Babbage (last era that runs `NEWPP` rule), there is a chance of overflow here. It is safer to convert to Integer. Here is another example of dangerous usage of `fromIntegral`
```suggestion
      if (toInteger (ppNew' ^. ppMaxTxSizeL) + toInteger (ppNew' ^. ppMaxBHSizeL) < toInteger (ppNew' ^. ppMaxBBSizeL))
```

Reply:
done